modules/ROOT/pages/create-community.adoc

```diff
@@ -103,13 +103,45 @@ To verify that Salesforce CRM Content is enabled:
 
 After you create a community, you must create different user profiles for your community _members_ (signed in) and _guests_, so that you can control access to and visibility of APIs and content.
 
+Before configuring member and guest profiles and permissions, open *Setup* > *User Management Settings* and ensure that *Enhanced Profile User Interface* is enabled.
+
 === Configure Member User Profile Permissions
 
-These are the permissions for the member users:
+To configure permissions for the member user profile so your members can access APIs, create client applications, and manage their details:
+
+. In *Setup*, search for *Users* using the *Quick Find* box and click *Users/Profiles*.
+. Click the *Customer Community Plus Login User* profile.
+. Click *Clone*.
+. For *Profile Name* enter `[Site Name] Member User`.
+. Click *Save*.
+. In the *[Site Name] Member User* profile page, click *Edit*.
+. In *Object Settings*, edit the object permissions to match those shown in the following table.
+. Click *Save*.
+. Return to the profile overview.
+. Click the link for *Apex Class Access*.
+. Click *Edit*.
+. Select these classes in the list and click *Add* to make them accessible:
+.. `acm_pkg.AnypointApiAnalyticsController`
+.. `acm_pkg.ApiAccessRequester`
+.. `acm_pkg.ApiPortalController`
+.. `acm_pkg.ApiVersionHeader`
+.. `acm_pkg.CustomFieldsViewerController`
+.. `acm_pkg.FineGrainVisibility`
+.. `acm_pkg.LightningForgotPasswordController`
+.. `acm_pkg.LightningLoginFormController`
+.. `acm_pkg.LightningSelfRegisterController`
+.. `acm_pkg.QueryApiController`
```

Comment:
If these are UI choices selected from a UI list, they should be bold.
Take a look at the style ref. entries for "bold font" and for "monospace font," because I'm having trouble envisioning this UI?

Reply:
These are choices from a list, and I've updated them to be bold.